### PR TITLE
Improvements for Payload inspector

### PR DESCRIPTION
**Pull request by ggovi**
#### PR description:

Payload Inspector Interface generalised to support combinations of iov-policies and tag multiplicity 

#### PR validation:

- All the System-specific unit tests executed; picture plot files compared with previous release after generation
- BasicPayload plots generated in stand-alone running the getPayloadData script
- Current branch deployed as the DEMO release  in the Payload Inspector application

### CODE REVIEW

**Comment by ggovi** [comment]
please test

File: CondCore/Utilities/interface/PayloadInspector.h [hunk 1]
```diff
@@ -3,14 +3,37 @@
 
 #include "CondCore/CondDB/interface/Utils.h"
 #include "CondCore/CondDB/interface/Session.h"
-#include "FWCore/Utilities/interface/GlobalIdentifier.h"
```
Comment (mmusich) [inline CondCore/Utilities/interface/PayloadInspector.h:6]: @ggovi 
ins't this undoing commit https://github.com/cms-sw/cmssw/pull/29221/commits/3fb4a123c78b9dc0b6f475a4bf783af5607a309b from PR #29221 ?

**Comment by silviodonato** [comment]
please test

**Comment by ggovi** [comment]
please test

**Comment by ggovi** [comment]
please test

**Comment by ggovi** [comment]
+1

**Comment by ggovi** [comment]
@Martin-Grunewald @fwyzard
any comment/feedback? ( not sure what is the relevance for HLT here... )

**Comment by Martin-Grunewald** [comment]
+1
me neither....

**Comment by fwyzard** [comment]
+1

I guess it is because of `CondCore/HLTPlugins`, the changes look in line with the rest.

**Comment by silviodonato** [comment]
+1

**Pull request merged**